shell/client/styles/_topbar.scss

```diff
@@ -62,6 +62,56 @@ body>.topbar-update {
   }
 }
 
+body>li.navitem-grain {
```

Comment:
I'm curious why this block is needed.  Does `dragula` just move the original grabbed DOM node to the root of the document while it's in motion?